### DISCUSSION TITLE
Horizontally Stacked containers

### DISCUSSION THREAD

**Discussion by Samrose-Ahmed** [cloudscape-design/components#1020] [Q&A] [answered]
Is there a simple way to compose containers horizontally. The `variant='stacked'` prop on Container helps with vertical containers but how about left to right containers (analagous to `flex-direction:row`.

**Comment by jperals** [comment]
Hello,

Would the Grid component help you here?

See:
- [Grid component](https://cloudscape.design/components/grid)
- [Example CodeSandbox](https://codesandbox.io/s/hungry-euclid-dkl4n5?file=/src/App.js)

**Reply by Samrose-Ahmed** [reply]
That's not the same thing, think of layouts like for example this (Athena) :

<img width="819" alt="Screenshot 2023-05-07 at 7 14 04 PM" src="https://user-images.githubusercontent.com/9027301/236718942-a3f2f7bf-1f1e-4c67-b3e4-c10a5fb67864.png">

You can try to use `disableGutters` on `Grid`, but its not right since the border and shadows need to removed to look integrated

<img width="417" alt="Screenshot 2023-05-07 at 7 19 10 PM" src="https://user-images.githubusercontent.com/9027301/236719401-a62afb35-acfe-4c85-bc55-00e78529dbef.png">


(which is what `variant='stacked'` seems to do nicely for vertically laid out containers).

<img width="567" alt="Screenshot 2023-05-07 at 7 20 34 PM" src="https://user-images.githubusercontent.com/9027301/236719591-88cdaea7-b0da-4bfd-8f51-7d86d5aee894.png">

**Comment by gethinwebster** [comment, marked as answer]
We don't currently have the functionality to stack containers with no gaps like this, we'd recommend leaving a small gap between the containers to take account for the rounded corners and shadows that are present in the open source styled components.
